### PR TITLE
Feature/change fee index slider

### PR DESCRIPTION
**Pull request by alexjavabraz**

### CODE REVIEW

File: src/components/create/BtcFeeSelect.vue [hunk 1]
```diff
@@ -135,5 +137,24 @@ export default class BtcFeeSelect extends Vue {
     }
     this.setSelectedFee(selectedFee);
   }
+
+  beforeMount() {
+    let selectedFee = 1;
+    if (this.pegInTxState && this.pegInTxState.selectedFee) {
+      console.log('Verifying switch...');
```
Comment (ronaldsg20) [inline src/components/create/BtcFeeSelect.vue:144]: Delete this log please

**Review by ronaldsg20 — APPROVED** [review]

File: src/components/create/BtcFeeSelect.vue (hunk 1 above)
Comment (alexjavabraz) [reply src/components/create/BtcFeeSelect.vue]: removed, thank you.

**Review by hard75 — APPROVED** [review]
checked fee saved

**Review by ronaldsg20 — APPROVED** [review]

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Pull request merged**